src/app/src/main/kotlin/org/imperial/mrc/hint/security/SecurityConfig.kt

```diff
@@ -1,17 +1,45 @@
 package org.imperial.mrc.hint.security
 
+import org.pac4j.core.client.Clients
 import org.pac4j.core.config.Config
+import org.pac4j.core.context.session.J2ESessionStore
+import org.pac4j.http.client.indirect.FormClient
 import org.pac4j.springframework.web.SecurityInterceptor
+import org.pac4j.sql.profile.service.DbProfileService
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.ComponentScan
 import org.springframework.context.annotation.Configuration
+import org.springframework.jdbc.datasource.TransactionAwareDataSourceProxy
 import org.springframework.web.servlet.config.annotation.InterceptorRegistry
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer
+import javax.sql.DataSource
 
 @Configuration
-class SecurityConfig(val config: Config): WebMvcConfigurer {
+@ComponentScan(basePackages = ["org.pac4j.springframework.web", "org.pac4j.springframework.component"])
+class Pac4jConfig {
```

Comment:
So much for Beans outside classes - this has to be a class because it needs ComponentScan...